app/poros/github_users.rb

```diff
@@ -1,6 +1,10 @@
 class GithubUsers
+  def initialize(token)
+    @token = token
```

Comment:
I thought this was the cleanest way, instead of passing a token argument to each of the methods below

Reply:
yep, I had the same thought!